### DISCUSSION TITLE
Apple and Google pay support on web

### DISCUSSION THREAD

**Discussion by racingDeveloper** [flutter-stripe/flutter_stripe#1132] [Q&A]
Hi!

I was just curious to understand if you are planning sooner or later to support apple and google pay for the flutter_web version.

I am working on a project that is relying a lot on flutter web with stripe, and we would like to add Apple and Google pay.

Thank you in advance,
Gabriele

**Comment by nokia6290** [comment]
Did u got working stripe for flutter web in production?
